include/seqan3/alphabet/quality/all.hpp

```diff
@@ -61,17 +61,42 @@
+ *
+ * The following figure gives a graphical explanation of the different Alphabet Types:
+ *
+ * <span style="font-size:12px;font-family:Courier New;line-height:1;">
+        <span style="color:DarkSlateBlue"> SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS....................................................</span>\n
+        <span style="color:Chartreuse">    MMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMMM...............................</span>\n
+        <span style="color:Chocolate">     PPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPPP</span>\n
+        <span style="color:BlueViolet">    ..........................OOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO</span>\n\n
+        <span style="font-weight:bold"> !\"\#\$\%\&'()*+,-\./0123456789:;\<\=\>?\@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\\]^_`abcdefghijklmnopqrstuvwxyz{\|}~</span>\n
+                                       \|.........................\|..............\|....................\|..............................\|\n
+                                           33........................59.............74...................95............................126\n\n
+        <span style="color:DarkSlateBlue"> 0________________________________________40....................................................</span>\n
+        <span style="color:Chartreuse">    0________________________________________40___________________62...............................</span>\n
+        <span style="color:Chocolate">     0________________________________________40___________________62_____________________________93</span>\n
+        <span style="color:BlueViolet">    .........................-5____0_________9____________________62_____________________________93</span>\n\n
+        <span style="color:DarkSlateBlue"> S - Sanger, Illumina 1.8+ - phred42</span>\n
+        <span style="color:Chartreuse">    M - Sanger, Illumina 1.8+ - phred63</span>\n
+        <span style="color:Chocolate">     P - Sanger, Illumina 1.8+ - phred94 (PacBio)</span>\n
+        <span style="color:BlueViolet">    O - Solexa - phred68legacy</span>\n
+   </span>
+ *
+ * Normally, quality values come together with nucleotides. Therefore, it is obvious to combine those alphabets what we
+ * can do via seqan3::qualified. With seqan3::qualified, the cross product between the alphabets is formed, which leads
+ * to the main advantage: compression.
+ * The following combinations all still fit into a single byte:
+ * - `seqan3::qualified<seqan3::dna4, seqan3::phred42>` (alphabet size: 4 x 42 = 168)
+ * - `seqan3::qualified<seqan3::dna4, seqan3::phred63>` (alphabet size: 4 x 63 = 252)
+ * - `seqan3::qualified<seqan3::dna5, seqan3::phred42>` (alphabet size: 4 x 42 = 210)
+ * A vector over such a combined alphabet can thus save 50% of storage space compared to two separate vectors or using
+ * seqan3::phred94. The same combinations are of course with RNA alphabets possible.
```

Comment:
```suggestion
 * Quality values are usually paired together with nucleotides. Therefore, it stands to reason to combine both alphabets
 * into a new data structure. In SeqAn, this can be done with seqan3::qualified. It represents the cross product between
 * a nucleotide and a quality alphabet and is the go-to choice when compression is of interest.
 * The following combinations still fit into a single byte:
 * - `seqan3::qualified<seqan3::dna4, seqan3::phred42>` (alphabet size: 4 x 42 = 168)
 * - `seqan3::qualified<seqan3::dna4, seqan3::phred63>` (alphabet size: 4 x 63 = 252)
 * - `seqan3::qualified<seqan3::dna5, seqan3::phred42>` (alphabet size: 4 x 42 = 210)
 * Using `seqan3::qualified` can half the storage usage compared to storing qualities and nucleotides separately. Note
 * that any combination of `seqan3::phred94` with another alphabet will cause  `seqan3::qualified` use at least 2 bytes. 
 * While we used DNA alphabets in this example, the same properties hold true for RNA alphabets.
```

?